apps/passport-client/components/modals/FrogCryptoUpdateTelegramModal.tsx

```diff
@@ -60,7 +60,7 @@ export function FrogCryptoUpdateTelegramModal({
               appConfig.zupassServer,
               {
                 pcd: await credentialManager.requestCredential({
-                  signatureType: "sempahore-signature-pcd"
+                  signatureType: "semaphore-signature-pcd"
```

Comment:
this is somewhat dangerous, have you thought through the implications?

Reply:
Whoah, crazy that this typo made it into a PCD type name.  Changing that is definitely dangerous.  If anyone has any of these saved in their PCDCollection (like I do) I'm pretty sure it'll crash on load.

It may be best to just leave the typo in place in the cases where it's part of a data format.  Or if we want to fix it, I think we'd need to introduce some notion of type aliases and/or conversion to the PCD framework.

Reply:
For context this is not a pcd type name, this is the name of something only available in the client - a type of 'credential'

Reply:
Oh, so it is.  I incorrectly assumed there was a 1:1 mapping.  Seems like this is a field in a type with only one value, which is sent but never read.  So maybe the compatibility impact is minimal.